.github/workflows/publish-packages.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Packages
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '*/30 * * * *' # every 30 minutes
+
+# publishing can take longer than 30 minutes but we never want to
+# run more than one publish job at a time
+concurrency: 1
```

Comment:
FWIW this might be misleading; this option is effectively the name of the lock, not a number.

Reply:
I assumed it was one or the other and the result would be the same 😆